src/Database/Query/Processors/As400Processor.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace DreamFactory\Core\As400\Database\Query\Processors;
+
+use DreamFactory\Core\As400\Database\Query\Grammars\As400Grammar;
+use Illuminate\Database\Query\Builder;
+use Illuminate\Database\Query\Processors\Processor;
+
+class As400Processor extends Processor
+{
+    /**
+     * Process an "insert get ID" query.
+     *
+     * @param \Illuminate\Database\Query\Builder $query
+     * @param string $sql
+     * @param array $values
+     * @param string $sequence
+     *
+     * @return int/array
+     */
+    public function processInsertGetId(Builder $query, $sql, $values, $sequence = null)
+    {
+        $sequenceStr = $sequence ?: 'id';
+
+        if (is_array($sequence)) {
+            $grammar = new As400Grammar;
+            $sequenceStr = $grammar->columnize($sequence);
+        }
+
+        $sqlStr = 'select %s from new table (%s)';
```

Comment:
@anas-srikou Could you please explain why we are using this sql string? Just curious what `new table` means here. Thanks!

Reply:
@krishnapriawan 
This `new` keyword is used to get the new version of the table after inserting a new record in it from what I understand.
It's the same code used in the DB2 package, the code was just updated for readability

Reply:
@anas-srikou Got it, thanks for explaining!